src/components/Dropdown/props.ts

```diff
@@ -118,4 +127,24 @@ export type DropdownProps = {
    * list of triggers that can open the Dropdown (accepts: click, hover, contextMenu).
    */
   triggers?: DropdownTrigger[],
+
+  /**
+   * Called when the open state changes. Not triggered when hidden by click item.
+   *
+   * @param open the current open state of the Dropdown
+   * @param info an object containing the `source` of the open change, that can be:
+   * - trigger: the open state was changed by an external trigger
+   *    (e.g. the dropdown is opening because the children was clicked, or closing because the focus was lost)
+   * - menu: the open state was changed by clicking on a menu item
+   *    (e.g. the dropdown is closing because a menu item was clicked)
+   * @returns
+   */
+   onOpenChange?: (open: boolean, info?: OpenChangeInfo) => void
+
+   /**
+    * To set the container of the dropdown menu.
+    * The default is to create a div element in body,
```

Comment:
I didn't get this sentence, you mean that by default the dropdown div will be appended in the body?

Reply:
That's what Antd documentation says.
 By inspecting the element, I actually observed the default behavior is to actually append it to the body elements, i.e. at the root level right before the `</body>` closing tag.

Reply:
ok, I was more worried about the phrasing clearness tbh